strings-csharp/StringFormattableString/StringFormattableString/FormattableStringsMethods.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+
+namespace StringFormattableString
+{
+    [MemoryDiagnoser]
+    [RankColumn]
+    [Orderer(SummaryOrderPolicy.FastestToSlowest)]
+    public class FormattableStringsMethods
+    {
+        [Benchmark]
+        public string StringExample() 
+        {
+            var studentName = "John";
+            var studentAge = 30;
```

Comment:
Since these values are shared between both benchmarks, and are constant strings, we could make them member constants of the benchmark class.

Reply:
@martinchege324 it looks like this one hasn't been addressed.

Reply:
Sorry indeed you have. I'm happy with this now.

Reply:
> @PhilBroderick I used different values for both methods; let me know if this change is still required.

Since this is a benchmark we shouldn't be using different values for the interpolated string. It should be the same between the two benchmarks. It would be better to move these to class constants and then use the class constants within the benchmark methods.